extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -309,9 +341,74 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
   }
 
   /** @override */
+  buildCallback() {
+    super.buildCallback();
+
+    if (this.isResponsive() && !this.responsiveSizeChangePromise_) {
```

Comment:
Would you expect build callback to be called more than once?  Otherwise why have the !this.responsiveSizeChangePromise_ check?  Assuming build callback does only occur once then you can remove responsiveSizeChangePromise_ as a field